code/modules/client/preference_setup/loadout/items/head.dm

```diff
@@ -529,6 +227,104 @@
 	caps["peaked cap, Private Military Contracting Group, no logo"] = /obj/item/clothing/head/peaked_cap/pmcg/no_logo
 	gear_tweaks += new /datum/gear_tweak/path(caps)
 
+/datum/gear/head/warden
+	display_name = "headwear, security (warden)"
+	description = "A selection of warden headwear."
+	path = /obj/item/clothing/head
+	allowed_roles = list("Head of Security" , "Warden", "Security Personnel")
+
+/datum/gear/head/warden/New()
+	..()
+	var/list/wardenhead = list()
+	wardenhead["warden hat, zavod"] = /obj/item/clothing/head/warden/zavod
+	wardenhead["warden hat, zavod alt"] = /obj/item/clothing/head/warden/zavod/alt
+	wardenhead["warden hat, idris"] = /obj/item/clothing/head/warden/idris
+	wardenhead["warden hat, pmc"] = /obj/item/clothing/head/warden/pmc
+	wardenhead["warden beret"] = /obj/item/clothing/head/beret/security/warden
```

Comment:
in another place you changed "beret, SCC" to "SCC beret" --> it should then be "zavodskoi warden hat"  and etc. here, no?

Reply:
i'm using beret, SCC as the standard now - the rest of the loadout items to follow, just the hats for now